packages/frontend-v2/components/Plan/Plan.tsx

```diff
@@ -11,11 +11,13 @@ import { useState } from "react";
 import { addClassesToTerm, removeYearFromPlan } from "../../utils";
 import { removeCourseFromTerm } from "../../utils/";
 import { ScheduleYear } from "./ScheduleYear";
+import { useDroppable } from "@dnd-kit/core";
 
 interface PlanProps {
   plan: PlanModel<string>;
   preReqErr?: PreReqWarnings;
   coReqErr?: CoReqWarnings;
+  setIsRemove?: (val: boolean) => void
```

Comment:
This is passed all the way down to the schedule course that is being rendered. This is the differentiator between the initial schedule course, and the one that DND kit creates when you start dragging. We use this `isRemove` value and `setIsRemove` to essentially tell react that **this schedule course** is the one that is being dragged, and if it is not over a droppable, style it a specific way to indicate to the user that it is being removed.